tests/functional/test_zilien_reader.py

```diff
@@ -259,6 +260,13 @@ def test_series_match(self, datasets):
             for mpt_name in mpt.series_names:
                 if mpt_name == "time/s":
                     assert f"Biologic {mpt_name}" in tsv.series_names
+                elif mpt_name.endswith("=0"):
+                    # ConstantValue series with "=0" suffix is created when an
+                    # essential series is not in the parsed dataset.
+                    # This suffix is removed by Zilien when integrating .mpt files,
```

Comment:
@matenestor , what is Zilien's behavior when running with EC data incorporated from a Biologic file with `Ns`? Without `Ns`?
When does Zilien add an Ns column to the data? Does it remove a "=0" suffix, or just (in contrast to ixdat) not add such a suffix?

Reply:
@ScottSoren Zilien will now remove the "=0" suffix created from all columns names containing it, so they are there in a pure form, since ixdat doesn't load the columns as `ConstantValue` anyway.
Though, there is a column `raw_current=0` (at least this one) being produced by Zilien, because back then I didn't think there would be a difference. But I am patching the dataset generation now.